src/codegen/mod.rs

```diff
@@ -793,8 +793,64 @@ impl CodeGenerator for Type {
                             "repr_transparent feature is required to use {:?}",
                             alias_style
                         );
+
+                        let mut attributes = vec![attributes::repr("transparent")];
+                        let mut derives = vec![];
+
+                        if item.can_derive_debug(ctx) {
+                            derives.push("Debug");
+                        }
+
+                        if item.can_derive_default(ctx) {
+                            derives.push("Default");
+                        }
+
```

Comment:
Is there any chance we can share this code with the other code that build similar attribute lists?

Reply:
Sure—see latest commit.